scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -260,12 +260,84 @@ class ModelTaskName(Enum):
     TEXT_CLASSIFICATION = 'text-classification'
 
 
-class ModelDownloadType(Enum):
-    """Enum for Download Method for Remote Model."""
+class PathType(Enum):
+    """Enum for path types supported for model publishing."""
 
-    FTP = 'ftp'
-    GIT = 'git'
-    HTTP = 'http'
+    LOCAL = "local"  # Path to model files present locally.
+    GIT = "git"      # Model hosted on a public GIT repo and can be cloned by GIT LFS.
+    FTP = "ftp"      # <UNSUPPORTED> Model files hosted on a FTP endpoint.
+    HTTP = "http"    # <UNSUPPORTED> Model files hosted on a HTTP endpoint.
+    AZURE = "azure"  # Model files hosted on an AZURE blobstore with public read access.

```

Comment:
>AZURE = "azure"  # Model files hosted on an AZURE blobstore with public read access. [](http://example.com/codeflow?start=4&length=84)

Wondering if we want to rename this to `azureblob` as we will add support for more data stores in the future.

Reply:
https://azure.microsoft.com/en-in/products/

![image](https://user-images.githubusercontent.com/482709/204199932-9bb10496-24f8-49b0-b120-747fe77f911e.png)

Reply:
Make sense. Modified to azureblob